src/v2/Apps/Artist/Routes/Overview/Utils/renderOrRedirect.tsx

```diff
@@ -20,39 +19,10 @@ export function renderOrRedirect({
     return undefined
   }
 
-  const showMarketInsights = () => {
-    // Is there a gallery representation section? Is there an auction highlights
-    // section? Is there a permanent collections section?
-    if (
-      artist?.artistPartnersConnection?.edges?.length ||
-      artist?.auctionResults?.edges?.length ||
```

Comment:
This seems to be always undefined because `auctionResults` doesn't exist on `artist` at this point.